res/openvpn-gui-res-zh-hans.rc

```diff
@@ -327,12 +327,12 @@ BEGIN
     IDS_ERR_UNKNOWN_PRIORITY "未知的优先权名称: %ls"
     IDS_ERR_LOG_APPEND_BOOL "追加日志文件的标志（例如「%ls」）必须为「0」或「1」"
     IDS_ERR_GET_MSIE_PROXY "无法获取 MSIE 的代理设置。"
-    IDS_ERR_INIT_SEC_DESC "InitializeSecurityDescriptor 失败。"
-    IDS_ERR_SET_SEC_DESC_ACL "SetSecurityDescriptorDacl 失败。"
-    IDS_ERR_CREATE_PIPE_OUTPUT "CreatePipe hOutputWrite 失败。"
-    IDS_ERR_CREATE_PIPE_INPUT "CreatePipe hInputRead 失败。"
-    IDS_ERR_DUP_HANDLE_OUT_READ "DuplicateHandle on hOutputRead 失败。"
-    IDS_ERR_DUP_HANDLE_IN_WRITE "DuplicateHandle on hInputWrite 失败。"
+    IDS_ERR_INIT_SEC_DESC "初始化安全描述符失败。"
+    IDS_ERR_SET_SEC_DESC_ACL "设置安全描述符acl失败。"
+    IDS_ERR_CREATE_PIPE_OUTPUT "创建管道输出写入失败。"
+    IDS_ERR_CREATE_PIPE_INPUT "创建管道输入读取失败。"
+    IDS_ERR_DUP_HANDLE_OUT_READ "输出读取时的重复句柄失败。"
+    IDS_ERR_DUP_HANDLE_IN_WRITE "输入写入时的重复句柄失败。"
```

Comment:
Shouldn't Windows API function names like "InitializeSecurityDescriptor", "DuplicateHandle" etc. should be  untranslated as in the original?

Reply:
> 
Sorry, I shouldn't translate here.
Thank you for reminding me

Reply:
You seem to have closed this. You could push a fixup commit which we can squash during merge. Or do you plan to submit a new PR with updated commits? There is some apparently good stuff in here.